packages/webdriverio/tests/commands/element/getValue.test.js

```diff
@@ -20,46 +20,19 @@ describe('getValue', () => {
 
         await elem.getValue()
         expect(request.mock.calls[2][0].uri.path).toBe('/wd/hub/session/foobar-123/element/some-elem-123/property/value')
-        expect(request.mock.calls[3]).toBeFalsy()
     })
 
     test('should get the value using getElementAttribute', async () => {
         const tmp = await browser.$('#foo')
         const elem = {
             elementId : 123,
             getValue : tmp.getValue,
-            getElementProperty : jest.fn(() => {
-                throw {
-                    message : 'this.getElementProperty is not a function',
-                }
-            }),
+            isW3C : false,
```

Comment:
you can emulate this case by creating a browser instance with browser name ending with `-noW3C`  like in this test https://github.com/webdriverio/v5/blob/master/packages/webdriverio/tests/commands/element/addValue.test.js#L28